packages/portal-proto/src/components/SearchInput.tsx

```diff
@@ -145,32 +281,40 @@ export const SearchInput: React.FC = () => {
             </p>
           ) : (
             <>
-              <P>Related Categories</P>
-              <div className="flex flex-wrap gap-1 my-2">
-                {uniq(searchResults.map((result) => result.category)).map(
-                  (cat) => {
-                    const selected = filteredCategories.includes(cat);
-                    return (
-                      <Badge
-                        className={
-                          selected
-                            ? "text-white bg-primary-dark capitalize text-sm font-normal hover: "
-                            : "text-primary-dark border-solid border-primary-dark capitalize text-sm font-normal hover:text-white hover:bg-primary-dark"
-                        }
-                        color="white"
-                        tabIndex={0}
-                        onClick={() => toggleCategory(selected, cat)}
-                        onKeyDown={createKeyboardAccessibleFunction(() =>
-                          toggleCategory(selected, cat),
-                        )}
-                        leftSection={selected ? <CheckIcon /> : undefined}
-                        key={cat}
-                      >
-                        {cat}
-                      </Badge>
-                    );
-                  },
-                )}
+              <div
+                role="group"
+                aria-label="Filter Results by Related Categories"
+                aria-controls={comboboxlistId}
+              >
+                <P>Related Categories</P>
+                <ul className="flex flex-wrap gap-1 my-2 gap-y-0">
```

Comment:
Could this just be `gap-x-1` instead of `gap-1 gap-y-0`?

Reply:
was just copying code good catch